### ISSUE TITLE
DFSR Collector "counter not found"

### ISSUE THREAD

**Issue by davideverall** [prometheus-community/windows_exporter#1594]
### Current Behavior

When running windows_exporter 0.27.1/0.27.2 with dfsr collector enabled, an event error is logged every 30 seconds, although counters appear to be working. Error message:

`ts=2024-08-30T10:15:03.024Z caller=prometheus.go:186 level=error msg="collector dfsr failed after 0.000000s" err="counter not found"`

### Expected Behavior

There should be no error running the collector

### Steps To Reproduce

```Markdown
1. Run windows exporter 0.27.1/0.27.2 with dfsr collector enabled & "volume" Perflib sources enabled
2. Check windows application logs for errors
```

### Environment

- windows_exporter Version: 0.27.2
- Windows Server Version: Server 2019 Standard

### windows_exporter logs

```shell
C:\Program Files\windows_exporter>.\windows_exporter.exe --collectors.enabled "dfsr" --collectors.dfsr.sources-enabled=volume --log.level=debug
ts=2024-08-30T10:14:46.565Z caller=exporter.go:147 level=debug msg="Logging has Started"
ts=2024-08-30T10:14:46.565Z caller=wmi.go:16 level=debug msg="Initializing SWbemServices"
ts=2024-08-30T10:14:46.580Z caller=dfsr.go:163 level=info collector=dfsr msg="dfsr collector is in an experimental state! Metrics for this collector have not been tested."
ts=2024-08-30T10:14:46.690Z caller=exporter.go:222 level=info msg="Running as TNP\\everalld-da"
ts=2024-08-30T10:14:46.690Z caller=exporter.go:229 level=info msg="Enabled collectors: dfsr"
```

ts=2024-08-30T10:14:46.692Z caller=exporter.go:264 level=info msg="Starting windows_exporter" version="(version=0.27.2, branch=HEAD, revision=4e460bc24c3668fbef641a187162073cd5da2ecb)"
ts=2024-08-30T10:14:46.692Z caller=exporter.go:265 level=info msg="Build context" build_context="(go=go1.22.6, platform=windows/amd64, user=runneradmin@fv-az980-61, date=20240817-19:03:46, tags=unknown)"
ts=2024-08-30T10:14:46.692Z caller=exporter.go:266 level=debug msg="Go MAXPROCS" procs=4
ts=2024-08-30T10:14:46.693Z caller=tls_config.go:313 level=info msg="Listening on" address=[::]:9182
ts=2024-08-30T10:14:46.694Z caller=tls_config.go:316 level=info msg="TLS is disabled." http2=false address=[::]:9182
ts=2024-08-30T10:15:03.024Z caller=prometheus.go:186 level=error msg="collector dfsr failed after 0.000000s" err="counter not found"
ts=2024-08-30T10:15:33.023Z caller=prometheus.go:186 level=error msg="collector dfsr failed after 0.000000s" err="counter not found"
```


### Anything else?

With "volume" Perflib sources disabled, there is no error. e.g.

```
C:\Program Files\windows_exporter>.\windows_exporter.exe --collectors.enabled "dfsr" --collectors.dfsr.sources-enabled=folder,connections --log.level=debug

ts=2024-08-30T10:16:33.043Z caller=prometheus.go:189 level=debug msg="collector dfsr succeeded after 0.017625s."

```

I have successfully [rebuilt performance counters](https://learn.microsoft.com/en-us/troubleshoot/windows-server/performance/manually-rebuild-performance-counters#rebuild-all-performance-counters-including-extensible-and-third-party-counters) as suggested but this did not resolve the errors.

**Comment by jkroepke** [comment]
Could you please execute this command on cmd:

```
typeperf -q | findstr DFS
```

**Comment by davideverall** [comment]
Hi

The output is:

C:\Windows\system32>typeperf -q | findstr DFS
\DFS Namespace(*)\Folder Count
\DFS Namespace Service Referrals(*)\Requests Processed /sec
\DFS Namespace Service Referrals(*)\Requests Failed
\DFS Namespace Service Referrals(*)\Avg. Response Time
\DFS Namespace Service Referrals(*)\Requests Processed
\DFS Namespace Service API Requests(*)\Requests Processed /sec
\DFS Namespace Service API Requests(*)\Requests Failed
\DFS Namespace Service API Requests(*)\Avg. Response Time
\DFS Namespace Service API Requests(*)\Requests Processed
\DFS Replication Service Volumes(*)\Database Lookups
\DFS Replication Service Volumes(*)\Database Commits
\DFS Replication Service Volumes(*)\USN Journal Unread Percentage
\DFS Replication Service Volumes(*)\USN Journal Records Accepted
\DFS Replication Service Volumes(*)\USN Journal Records Read
\DFS Replication Connections(*)\Bandwidth Savings Using DFS Replication
\DFS Replication Connections(*)\RDC Bytes Received
\DFS Replication Connections(*)\RDC Compressed Size of Files Received
\DFS Replication Connections(*)\RDC Size of Files Received
\DFS Replication Connections(*)\RDC Number of Files Received
\DFS Replication Connections(*)\Bytes Received Per Second
\DFS Replication Connections(*)\Compressed Size of Files Received
\DFS Replication Connections(*)\Size of Files Received
\DFS Replication Connections(*)\Total Files Received
\DFS Replication Connections(*)\Total Bytes Received
\DFS Replicated Folders(*)\Bandwidth Savings Using DFS Replication
\DFS Replicated Folders(*)\RDC Bytes Received
\DFS Replicated Folders(*)\RDC Compressed Size of Files Received
\DFS Replicated Folders(*)\RDC Size of Files Received
\DFS Replicated Folders(*)\RDC Number of Files Received
\DFS Replicated Folders(*)\Compressed Size of Files Received
\DFS Replicated Folders(*)\Size of Files Received
\DFS Replicated Folders(*)\Total Files Received
\DFS Replicated Folders(*)\Deleted Space In Use
\DFS Replicated Folders(*)\Deleted Bytes Cleaned up
\DFS Replicated Folders(*)\Deleted Files Cleaned up
\DFS Replicated Folders(*)\Deleted Bytes Generated
\DFS Replicated Folders(*)\Deleted Files Generated
\DFS Replicated Folders(*)\Updates Dropped
\DFS Replicated Folders(*)\File Installs Retried
\DFS Replicated Folders(*)\File Installs Succeeded
\DFS Replicated Folders(*)\Conflict Folder Cleanups Completed
\DFS Replicated Folders(*)\Conflict Space In Use
\DFS Replicated Folders(*)\Conflict Bytes Cleaned up
\DFS Replicated Folders(*)\Conflict Files Cleaned up
\DFS Replicated Folders(*)\Conflict Bytes Generated
\DFS Replicated Folders(*)\Conflict Files Generated
\DFS Replicated Folders(*)\Staging Space In Use
\DFS Replicated Folders(*)\Staging Bytes Cleaned up
\DFS Replicated Folders(*)\Staging Files Cleaned up
\DFS Replicated Folders(*)\Staging Bytes Generated
\DFS Replicated Folders(*)\Staging Files Generated

**Comment by davideverall** [comment]
I can confirm that this is fixed in snapshot build 0.28.0-2
